pyproject.toml

```diff
@@ -40,7 +40,7 @@ packages = [
 
 [tool.poetry.dependencies]
 python = "^3.7"
-paho-mqtt = "^1.6"
+paho-mqtt = "1.5.1"
```

Comment:
Please never stick with a specific version. This results in all kind of issues for the community.
```suggestion
paho-mqtt = ">=1.5.1"
```

Reply:
(same comment as in the ospd-openvas repo):

> Please support newer versions of paho-mqtt too just use >=1.5.1

Currently the version 1.5.1 is in debian bullseye which has some serious bugs we have to work around. The on_disconnect function is not working in 1.5.1 resulting in a crash when the broker disconnects. We want to be aware of this while developing so we do not run into such problems again.

Reply:
We chose to do your suggestion of using ">=1.5.1" for now until a better solution is found.

Reply:
Thanks a lot!